### DISCUSSION TITLE
Help how to use Gopro Labs for video recording without using Logb

### DISCUSSION THREAD

**Discussion by NilsVega** [gopro/labs#1002] [General]
Hello to all!

I'm looking for settings for a GoPro Labs camera to shoot quality videos without using the LOGB profile. What resolution, FPS, color profile and other parameters would you recommend to get good results straight from the camera? If you have specific setup examples or tips, please share them.

Thanks for your help!

**Comment by dnewman-gpsw** [comment]
Moved from Issues (this is not a bug report.)

Without any context, the best recommendation is to use the camera defaults, they are already optimized for good results for most users.  Resolution and FPS depend on your application, and how you want to share.
